.gitignore

```diff
@@ -16,6 +16,7 @@ logs
 pids
 *.pid
 *.seed
+android/.graddle
```

Comment:
I understand that, so will this change actually remove those unneeded bin files added in the previous commit?